python/pyhail/keytable.py

```diff
@@ -0,0 +1,148 @@
+from pyhail.type import Type
+
+class KeyTable(object):
+    """:class:`.KeyTable` is Hail's version of a SQL table where fields
+    can be designated as keys.
+
+    """
+
+    def __init__(self, hc, jkt):
+        """
+        :param hc: Hail spark context.
```

Comment:
I think the convention is to put it all on one line if the type is a single word, and two lines if it is multiple words (e.g., T or S, list of int).  I made it one line.  I don't think `:class:` is necessary in type setting (rtype, param type field, type).  I will change it.